### PR TITLE
localize-behavior added to speakers-page.html

### PR DESCRIPTION
**Pull request by Alpccelik**
Hi 

I added Localize Behavior in speakers-page.html

### CODE REVIEW

File: data/uk/resources.json [hunk 1]
```diff
@@ -79,5 +79,6 @@
   "continue-reading": "Читати далі",
   "view-presentation": "Переглянути презентацію",
   "view-video": "Переглянути відео",
-  "share-with": "Поширити у"
+  "share-with": "Поширити у",
+  "skills":"навички"
```
Comment (ozasadnyy) [inline data/uk/resources.json]: Please add a space and start with capital letter
"skills": "Навички"

File: src/pages/speakers-page.html [hunk 2]
```diff
@@ -198,9 +199,9 @@
             </a>
             <a href$="/speakers/[[speaker.id]]/" class="card-content">
               <iron-image class="company-logo" src="{{speaker.companyLogo}}" sizing="contain"></iron-image>
-              <div class="tags">Skills:
+              <div class="tags">{{localize('skills')}}:
                 <template is="dom-repeat" items="[[speaker.tags]]" as="tag">
-                  <span class="tag" style$="{{_getCustomStyleValue(tag)}}" target="_blank" rel="noopener noreferrer">[[tag]]</span>
+                  <span class="tag" style$="{{_getCustomStyleValue(tag)}}" target="_blank">[[tag]]</span>
```
Comment (ozasadnyy) [inline src/pages/speakers-page.html]: Remove `target="_blank"` as well, my mistake :)

**Comment by ozasadnyy** [comment]
Looks amazing!
Few things to change and we will be able to merge :)

File: data/uk/resources.json (hunk 1 above)
Comment (Alpccelik) [inline data/uk/resources.json]: Done

File: src/pages/speakers-page.html (hunk 2 above)
Comment (Alpccelik) [inline src/pages/speakers-page.html]: done

**Comment by ozasadnyy** [comment]
Thanks for your contribution!

**Pull request merged**